src/builders/go/gomod2nix/default.nix

```diff
@@ -0,0 +1,21 @@
+{
+  lib,
+  pkgs,
+  externals,
+  ...
+}:
+
+{
+  fetchedSources,
+  dreamLock,
+}:
+let
+  gomod2nixTOML = fetchedSources.mapAttrs
+    lib.replaceStrings ["__slash__"] ["/"] dependencyObject.goName;
+in
+externals.gomod2nixBuilder rec {
+  pname = dreamLock.generic.mainPackage;
+  version = dreamLock.sources."${pname}".version;
+  src = fetchedSources."${pname}";
+  modules = ./gomod2nix.toml;
+}
```

Comment:
You should add en extra new line here so that git can generate diffs for the last line.

Reply:
Ohh didn't about this. Thanks!